sql/README.md

```diff
@@ -6,6 +6,8 @@
 
 > 本地DB中运行 sql/automl.sql 即可.
```

Comment:
Please also use k8s feature to execute db initial while MySQL pod has been created. Below is a guide about how to do it. https://medium.com/@AbhijeetKasurde/creating-mysql-pod-with-preloaded-database-2c01c002fdc3